examples/.bazelrc

```
@@ -24,3 +24,7 @@ test --build_tests_only
 # as the user configuration should be able to overwrite flags from this file.
 # See https://docs.bazel.build/versions/master/best-practices.html#bazelrc
 try-import %workspace%/user.bazelrc
+
+build --repo_env=CC=clang
+build --repo_env=CXX=clang++
+build --linkopt=-B/usr/bin/ld.lld
```

Please keep the .bazelrc files identical (CI fails because of this). Then we can see whether gcc build still works.

---

ah yep, this should be
```suggestion
build --linkopt=-fuse-ld=/usr/bin/ld.lld
```

---

The image's clang is installed in https://github.com/actions/runner-images/blob/main/images/linux/scripts/installers/clang.sh#L14,
so it does jsut install clang via apt, which means we have https://ubuntu.pkgs.org/22.04/ubuntu-universe-amd64/lld-14_14.0.0-1ubuntu1_amd64.deb.html
and the path `/usr/lib/llvm-14/bin/lld` may work.

---

OK CI is still on Ubuntu20.04, so I set it to the default lld-11 path.

---

```
lld is a generic driver.
Invoke ld.lld (Unix), ld64.lld (macOS), lld-link (Windows), wasm-ld (WebAssembly) instead
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
I guess that's not how that works then :sweat_smile:

---

Leaving it away does the trick. Interestingly, this does not work on my laptop, but I'm not running Ubuntu.